samples/nextjs/src/lib/page-props-factory.ts

```diff
@@ -114,16 +113,13 @@ export class SitecorePagePropsFactory {
           isServerSidePropsContext(context) ? (context as GetServerSidePropsContext).res : undefined
         )
         .catch((error) => {
-          if (error.isAxiosError) {
-            // RestLayoutService uses Axios by default
-            const axiosError = error as AxiosError<LayoutServiceData>;
-            if (axiosError.response?.status === 404) {
-              // Let 404s (invalid path) through for RestLayoutService.
-              // layoutData.sitecore.route will be missing, but
-              // layoutData.sitecore.context will provide valuable information
-              notFound = true;
-              return axiosError.response?.data;
-            }
+          if (error.response?.status === 404) {
```

Comment:
@ambrauer I noticed that you want to merge this branch into feature/461401.
In this PR you removed check for `isAxiosError`.
But in this PR: https://github.com/Sitecore/jss/pull/558/files#diff-43047d80ffb584ad1f0a1e953785a472f511e4404f4e8529e5bffe9a9bc6700cR117
you added it. So I'm a little bit confused, can you clarify it?

Reply:
@illiakovalenko Correct, this branch is dependent on #558. In that branch, I was still trying to achieve preservation of `layoutData.sitecore.context` which comes back with REST layout service (using Axios), while allowing GraphQL (different client) as well. However, in this PR, we no longer care to capture `layoutData.sitecore.context`, so this is simplified further.

Reply:
Got it